### DISCUSSION TITLE
Does Frigate support the new Alderlake CPU tensorflow acceleration?

### DISCUSSION THREAD

**Discussion by twctg81** [blakeblackshear/frigate#3306] [General]
Was reading an article on how the new alder lake cpus now can optimize tensorflow operations.
So does Frigate support those optimization for alder lake? 

https://medium.com/intel-analytics-software/accelerate-ai-model-performance-on-the-alder-lake-platform-a5c24ae3f522

**Comment by NickM-27** [comment]
Would have to read more about it specifically. If it's just dedicated hardware gates then it should come for free. Same as how Apple silicon performs really well.

**Comment by twctg81** [comment]
> [Intel Optimization for TensorFlow](https://www.intel.com/content/www/us/en/developer/tools/frameworks/overview.html#tensorflow) (version 2.8) provides up to 3.2x throughput performance improvements over the stock TensorFlow. Figures 1 and 2 show the performance improvements observed for common image recognition models from the [Intel AI Model Zoo](https://github.com/IntelAI/models).

Alright thanks.  The claim here is up to 3.2X throughput performance improvements so it's a nice alternative to the always out of stock google coral devices.
